editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1192,7 +1190,7 @@ void SpriteFramesEditor::_update_library(bool p_skip_selector) {
 
 	updating = true;
 
-	frame_duration->set_value(1.0); // Default.
+	frame_duration->set_value_no_signal(1.0); // Default.
```

Comment:
Should the same be done to `SpriteFramesEditor::_frame_list_item_selected`? Then you can probably remove the updating guards in it.

Reply:
> Then you can probably remove the updating guards in it.

Probably yes, but I wouldn't want to add any regressions. We'll probably refactor this plugin at some point and remove the obscurity in the code execution order.